src/main/scala/cromwell/engine/io/IOManager.scala

```diff
@@ -0,0 +1,38 @@
+package cromwell.engine.io
+
+import cromwell.binding.IoInterface
+
+//TODO: Use reflection to automagically implement all IOInterface methods ?
```

Comment:
Not sure about reflection, but there might be some possibilities with delegation/AOP macros.  Or at least this might be DRYer with the `findInterface(path) map { _.fn } getOrElse unsupported(path)` pulled out to a function.